### PR TITLE
Copilot Capabilities page string change

### PR DESCRIPTION
**Pull request by DenLilleMand**
#### Summary 
We are changing the strings and what links are shown depending on the users region in the Copilot Capabilities page

#### Work Item(s) 
Fixes [AB#537341](https://dynamicssmb2.visualstudio.com/Dynamics%20SMB/_workitems/edit/537341/)

### CODE REVIEW

File: src/System Application/App/AI/src/Copilot/CopilotAICapabilities.Page.al [hunk 1]
```diff
@@ -181,6 +180,9 @@ page 7775 "Copilot AI Capabilities"
                 AllowDataMovement := true;
         end;
 
+        ApplicationFamily := EnvironmentInformation.GetApplicationFamily();
+        IsWithinAlwaysOnRegion := ApplicationFamily.ToUpper() in ['US', 'GB', 'IN', 'AU'];
```
Comment (msft-sam) [inline src/System Application/App/AI/src/Copilot/CopilotAICapabilities.Page.al]: Is this different to `IsWithinGeo` from `ALCopilotFunctions`?

Comment (DenLilleMand) [reply src/System Application/App/AI/src/Copilot/CopilotAICapabilities.Page.al]: @msft-sam I changed it now to use the same conditionals as AlwaysConnected and NotAlwaysConnected, so these new groups visibility will toggle according to those.

File: src/System Application/App/AI/src/Copilot/CopilotAICapabilities.Page.al [hunk 2]
```diff
@@ -30,6 +30,19 @@ page 7775 "Copilot AI Capabilities"
     {
         area(Content)
         {
+            group(AllowedDataMovementOffInfo)
```
Comment (darjoo) [inline src/System Application/App/AI/src/Copilot/CopilotAICapabilities.Page.al:33]: Are we planning to have different text for when it's on/off?

**Review by darjoo — APPROVED** [review]

File: src/System Application/App/AI/src/Copilot/CopilotAICapabilities.Page.al (hunk 2 above)
Comment (DenLilleMand) [reply src/System Application/App/AI/src/Copilot/CopilotAICapabilities.Page.al:33]: @darjoo Yes.

On:
image


Off:
image

**Review by encimita — APPROVED** [review]

**Pull request merged**